client/state/signup/steps/website-content/schema.ts

```diff
@@ -0,0 +1,62 @@
+export const schema = {
+	type: 'object',
+	properties: {
+		currentIndex: 'number',
+		websiteContent: {
+			type: 'array',
+			items: { $ref: '#/$defs/pageData' },
+		},
+		$defs: {
+			pageData: {
+				type: 'object',
+				properties: {
+					title: {
+						type: 'string',
+					},
+					content: {
+						type: 'string',
+					},
+					images: {
+						type: 'array',
+						items: {
+							type: 'string',
+						},
+					},
+				},
+			},
+		},
+	},
+};
+
+export interface PageData {
+	title: string;
+	content: string;
+	images: Array< string >;
+}
+
+export type WebsiteContent = Array< PageData >;
+export interface WebsiteContentCollection {
+	currentIndex: 0;
```

Comment:
Should this line be this?
```suggestion
	currentIndex: number;
```